src/studying.md

```diff
@@ -53,97 +53,22 @@ you a little while to recall the answer, but as a general rule if you
 can’t answer within about 10 seconds, it’s probably better to move on
 and show the answer than keep struggling to remember.
 
-When the answer is shown, you should compare the answer you thought of
-with the answer which is shown and tell Anki how well you remembered. If
-you don’t trust yourself to compare your answer accurately, you can ask
-Anki to [prompt you to type in the answer](templates/fields.md#checking-your-answer) rather than
-just showing it to you.
+## Answer Buttons
 
-## Learning/Relearning Cards
+After the answer is shown, compare the answer you thought of
+with the answer which is shown and select any of the following button.
 
-When learning new cards, or when relearning cards that you have
-forgotten, Anki will show you the cards one or more times to help you
-memorize them. Each time is called a 'learning step'. By default, there
-are two steps: 1 minute and 10 minutes. You can change the number of
-steps and the delays between them in the [deck options](deck-options.md#new-cards).
+- **Again**: Select this when your answer is incorrect or when you couldn't recall the answer. If your answer is partially correct, you should be strict with yourself: if it counts as a fail in real-life context outside of Anki, then it counts as a fail in Anki as well. You'll typically use this button about 5-20% of the time. **Keyboard Shortcut**: <kbd>1</kbd>
```

Comment:
```suggestion
- **Again**: Select this when your answer is incorrect or when you couldn't recall the answer. If your answer is partially correct, you should be strict with yourself: if it counts as a fail in a real-life context outside of Anki, then it counts as a fail in Anki as well. You'll typically use this button about 5-20% of the time. **Keyboard Shortcut**: <kbd>1</kbd>
```

Reply:
Bolding 'keyboard shortcut' makes this section a bit difficult to read, as that repeated text has the same emphasis as again/hard/good/etc. How about a couple of newlines instead?